pkg/runtime/runtime.go

```diff
@@ -34,4 +44,14 @@ func init() {
 	default:
 		V1Mode, _ = strconv.ParseBool(v1Mode)
 	}
+
+	// You can switch the YAML library at runtime via an envvar:
+	switch os.Getenv(envvar.GoccyGoYaml) {
```

Comment:
Although goccy/go-yaml is the default for v1, you can opt in/out in both v0 and v1, by setting this envvar.